arelle/FileSource.py

```diff
@@ -342,6 +342,9 @@ def taxonomyPackageMetadataFiles(self):
         for f in (self.dir or []):
```

Comment:
Arbitrary line to allow reply for discussion:

> Side note: I'm not sure which parts of this constitutes the "reason for change" and which belong in the "description" section. I personally would add a "suggested change / fix" section

I think you nailed what we had in mind for the PR template at least, but open to suggestions to make it clearer. Our idea with the template is to get a description of what problem you are trying to solve (Reason for change), and then a second description of what you actually did to fix it (Description). I think "suggested change" makes a little more sense in the context of an issue but not as much sense when you have an open PR that's fixing the problem. Fix is similar I think, it implies that an issue/bug is being fixed but wouldn't make as much sense in the context of a new feature or improvement. Anyways I don't feel super strongly ultimately whatever is going to be easiest

Reply:
Agreed that suggestion makes more sense for issues. Perhaps the intent for PRs would be clearer if we titled the sections `Reason for change` and `Description of change`?